files/digitalocean/digitalocean_add_dbaas.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python
```

Comment:
what python version is needed? Also if we would run this script on clean VM what dependencies would be required?

Reply:
I tested it on python 2.7, which is what comes on the CentOS 7.9 image. I didn't install any extra modules, so it should run on a clean CentOS 7.9 VM. If the AMI version is using a later version of CentOS, I'd be glad to upgrade to python3, but I can't tell what the base CentOS version is on the centos-ami target now.